ibl_to_nwb/updated_conversion/brainwide_map/convert_brainwide_map.py

```diff
@@ -24,7 +27,12 @@ def convert_session(session: str, nwbfile_path: str):
         data_interfaces.append(StreamingIblRecordingInterface(session=session, stream_name=stream_name))
     for stream_name in lf_stream_names:
         data_interfaces.append(StreamingIblLfpInterface(session=session, stream_name=stream_name))
-    # TODO: initialize behavior and spike sorting interfaces
+    # TODO: initialize behavior and spike sorting
+
+    for camera_name in ["leftCamera", "rightCamera", "bodyCamera"]:
+        camera_files = one.list_datasets(eid=session, filename=f"*{camera_name}.dlc*")
+        if any("dlc" in x for x in camera_files):
```

Comment:
It seems to me that any filename returned by the previous line must have "dlc" in it, right? So couldn't this just be `if len(camera_files)`?

Reply:
Yeah my approach to this changed over time

The later PRs used a different iteration strat that instead fetches the available files, and then parses the variable camera names from those

Updated it here